hw9/ArrayValueCalculator.java

```diff
@@ -0,0 +1,29 @@
+package org.hillel.hw9;
+
+import org.hillel.hw9.exceptions.ArrayDataException;
+import org.hillel.hw9.exceptions.ArraySizeException;
+
+//Class calculates the sum of all elements of some String matrix/ Size of matrix should be 4x4
+//any element should contain any integer.
+//Сan throw exceptions if conditions are not met
+public class ArrayValueCalculator {
+    public static int doCalc(String[][] matrix){
+        for (int i = 0; i < matrix.length; i++) {
+            if (matrix.length != 4 || matrix[i].length != 4) {

```

Comment:
You should check `matrix.length != 4` only once. There is no need to do it in a loop. 
This check `matrix[i].length != 4` is correct and must be done in a loop, no questions here.